oracle/datadog_checks/oracle/oracle.py

```diff
@@ -104,26 +100,6 @@ def _fix_custom_queries(self):
                     if column.get('type') != 'tag':
                         column['name'] = '{}.{}'.format(prefix, column['name'])
 
-    def validate_config(self):
-        if not self._server or not self._user:
-            raise ConfigurationError("Oracle host and user are needed")
-
-        if not self._protocol or self._protocol.upper() not in VALID_PROTOCOLS:
-            raise ConfigurationError("Protocol %s is not valid, must either be TCP or TCPS" % self._protocol)
-
-        if self._jdbc_driver and self._protocol.upper() == PROTOCOL_TCPS:
```

Comment:
Why removing this validations if we are going to keep the jdbc?

Reply:
These should be moved to `config_models/validators`

Reply:
Got it. And since `py2` is no longer supported, we can delete them from here.